serde-oracle-jdbc-json/build.gradle.kts

```diff
@@ -0,0 +1,42 @@
+plugins {
```

Comment:
Rename to `serde-oracle-oson`

Reply:
It's not only OSON, it can parse ordinary JSON